exercises/practice/phone-number/HelloWorld.vb

```diff
@@ -0,0 +1,5 @@
+Public Class PhoneNumber
```

Comment:
I think this file should not be here (it's named `HelloWorld.vb`)

Reply:
Hmmm. Artefact. fixing